<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupButtonMenu</class>
 <widget class="QWidget" name="GroupButtonMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupButtonMenu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#background_{
	background-color: rgb( 90,90,90 );
}

/*group_new_mask_button_ */
QToolButton#group_new_mask_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#group_new_mask_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}
QToolButton#group_new_mask_button_:pressed{
	background-color: rgb(233, 111, 53);
	border-radius: 3px;
	border: 1px solid rgb(142, 67, 32);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}

/* group_delete_button_ */
QToolButton#group_delete_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#group_delete_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}
QToolButton#group_delete_button_:checked{
	background-color: rgb(233, 111, 53);
	border-radius: 3px;
	border: 1px solid rgb(142, 67, 32);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}

/* group_iso_button_ */
QToolButton#group_iso_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#group_iso_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}
QToolButton#group_iso_button_:checked{
	background-color: rgb(233, 111, 53);
	border-radius: 3px;
	border: 1px solid rgb(142, 67, 32);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}

/* duplicate_layer_button_ */
QToolButton#duplicate_layer_button_{
	background-color: rgba(0, 0, 0, 0);
	border: 1px solid rgba(0, 0, 0, 0);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}
QToolButton#duplicate_layer_button_:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}
QToolButton#duplicate_layer_button_:checked{
	background-color: rgb(233, 111, 53);
	border-radius: 3px;
	border: 1px solid rgb(142, 67, 32);
	margin: 0 0 0 0;
	padding: 0 0 0 0;
}


QWidget#group_tools_{
	border-radius: 4px;
	border: 1px solid rgb(90, 90, 90);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(161, 161, 161, 255), stop:0.155779 rgba(175, 175, 175, 255), stop:1 rgba(144, 144, 144, 255));
}

QWidget#buttons_{
	border-bottom: 1px solid rgb(90, 90, 90);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="background_" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="group_tools_" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="buttons_" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>27</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,1">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="duplicate_layer_button_">
              <property name="toolTip">
               <string>Duplicate a layer</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/DuplicateOff.png</normaloff>
                <normalon>:/Images/Duplicate.png</normalon>:/Images/DuplicateOff.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>35</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="group_iso_button_">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>22</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Open the isosurface menu</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/IsosurfaceMenuOff.png</normaloff>
                <normalon>:/Images/IsosurfaceMenu.png</normalon>:/Images/IsosurfaceMenuOff.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>35</width>
                <height>20</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="group_delete_button_">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Delete a layer</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/Delete.png</normaloff>
                <normalon>:/Images/DeleteWhite.png</normalon>:/Images/Delete.png</iconset>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="group_new_mask_button_">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Create a new mask layer</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../Resources/QtResources.qrc">
                <normaloff>:/Images/Add.png</normaloff>
                <normalon>:/Images/AddWhite.png</normalon>:/Images/Add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="iso_quality_" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_14" stretch="0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>4</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_19" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>16</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_31">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Isosurface Quality:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_18" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_24" stretch="1,1,1,1">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>1</number>
               </property>
               <item>
                <widget class="QRadioButton" name="radioButton_1_point_0">
                 <property name="text">
                  <string>1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_point_5">
                 <property name="text">
                  <string>.50</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_point_25">
                 <property name="text">
                  <string>.25</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_point_125">
                 <property name="text">
                  <string>.125</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="duplicate_layers_" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>32</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>12</number>
            </property>
            <property name="leftMargin">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="select_all_for_duplication_button_">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Select All</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="duplicate_button_">
              <property name="text">
               <string>Duplicate</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="delete_" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>32</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>32</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_25">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="select_all_button_">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip">
                  <string>Select all layers in this group</string>
                 </property>
                 <property name="text">
                  <string>Select All</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delete_button_">
                 <property name="text">
                  <string>Delete</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="facade_">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/QtResources.qrc"/>
 </resources>
 <connections/>
</ui>
